pkg/operator/k8sutil/network.go

```diff
@@ -116,7 +125,7 @@ func ApplyMultus(net cephv1.NetworkSpec, objectMeta *metav1.ObjectMeta) error {
 
 // getClusterNetworkApps returns the list of ceph apps that utilize cluster network
 func getClusterNetworkApps() []string {
-	return []string{"osd"}
+	return []string{"rook-ceph-osd"}
```

Comment:
@rohan47 is this going to break anything?

Reply:
@rohan47 up!

Reply:
@rohan47 bump